sycl/test-e2e/bindless_images/sampled_fetch/fetch_1D_USM.cpp

```diff
@@ -0,0 +1,104 @@
+// REQUIRES: linux
+// REQUIRES: cuda
+
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+
+#include <iostream>
+#include <sycl/sycl.hpp>
+
+// Uncomment to print additional test information
+// #define VERBOSE_PRINT
+
+class kernel_sampled_fetch;
+
+int main() {
+  sycl::device dev;
+  sycl::queue q(dev);
+  auto ctxt = q.get_context();
+
+  // Check if device supports 1D USM sampled image fetches
+  if (!dev.has(sycl::aspect::ext_oneapi_bindless_sampled_image_fetch_1d_usm)) {
+#ifdef VERBOSE_PRINT
+    std::cout << "Test skipped due to lack of device support for fetching 1D "
+                 "USM backed sampled images\n";
+#endif
+    return 0;
+  }
+
+  // declare image data
+  constexpr size_t width = 16;
+  std::vector<float> out(width);
+  std::vector<float> expected(width);
+  std::vector<float> dataIn(width);
+  auto imgMem = sycl::malloc_shared<float>(width, q);
+  for (int i = 0; i < width; i++) {
+    expected[i] = i;
+    imgMem[i] = i;
+  }
+
+  namespace syclexp = sycl::ext::oneapi::experimental;
+
+  try {
+    // Extension: image descriptor
+    syclexp::image_descriptor desc({width}, sycl::image_channel_order::r,
+                                   sycl::image_channel_type::fp32);
+
+    syclexp::bindless_image_sampler samp(
+        sycl::addressing_mode::repeat,
+        sycl::coordinate_normalization_mode::normalized,
```

Comment:
all the tests for 1D/2D/3D seem to use normalized coordinates.  Shouldn't we test for unnormalized as well?  And this would apply to the existing bindless_images/Sampling_1D/2D/3D tests as well.

I'm not asking for it as part of this PR, necessarily. Merely pointing it out.  Or maybe I'm overlooking something.

Reply:
I see "unnormalized" used in two of the other tests.  Will that be enough reasonable coverage?

Reply:
We can certainly add more comprehensive tests, however, I don't think it falls within the scope of this PR.

As far as the tests I've introduced here, the sampler is ignored when performing a "texture fetch", at least on the only backend we can test right now which is CUDA.

So whichever values are in the `bindless_image_sampler`, the fetch operation within the kernel will still be the same. I believe the intrinsics used for this in NVVM IR bypass the sampling hardware capabilities on NVidia devices. Our assumption is that this will be possible to implement on other backends as well.